server/server.go

```diff
@@ -9,6 +9,28 @@ import (
 	lru "github.com/hashicorp/golang-lru"
 )
 
+const usage = `<html><head><title>Not found</title></head><body>
```

Comment:
It'd be nice to display an emoji as favicon using this technique: https://css-tricks.com/emojis-as-favicons/

Maybe something like:
```html
<link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>👤</text></svg>">
```